doc/user/content/sql/create-sink.md

```diff
@@ -163,6 +165,22 @@ You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
 {{% kafka-sink-drop  %}}
 
+#### Enabling topic reuse after restart
+
```

Comment:
you can use the experimental tag like so `{{< experimental v0.8.2 />}}` similar to how the `SHOW CREATE TABLE` page does it

Reply:
then we don't need to write that its experimental only in the text